db/index.js

```diff
@@ -94,6 +94,32 @@ async function getUser({username, password}) {
   }
 }
 
+async function getAllUsers() {
+  try {
+      const { rows } = await client.query(`
+      SELECT *
+      FROM users;
+      `);
+      return rows;
+  } catch (error) {
+      throw error;
+  }
+}
+
+async function getUserById(id) {
+  try {
+      const { rows: [user] } = await client.query(`
+      SELECT *
+      FROM users
+      WHERE "id" = ${id};
```

Comment:
Hi Zach, I think for right here, the id doesn't need quotations because the serial id on the users tables doesn't have it. Just to be on the safe side, we could do the $1 to protect database from being broken from hackers so line 114 and 115 should be:
   WHERE id=$1;
        `, [ id ]); 

What are your thoughts, Laurence?

Reply:
Your'e right. It is now corrected.